src/content/lib/prefs.js

```diff
@@ -1,13 +1,23 @@
 // const Store = window.require('electron-store');
-import prefDefaults from "../../config/pref_defaults";
+const prefDefaults = {
+  // The user's bugzilla email
```

Comment:
`prefDefaults` was moved from the `pref_defaults.js` file (which was then deleted) and into `prefs.js` since that is the only place where `prefDefaults` is referenced.